### DISCUSSION TITLE
Update version 65. to latest version 110.

### DISCUSSION THREAD

**Discussion by senses10** [cefsharp/CefSharp#4431] [General]
sensesnewone@gmail.com
Could not load file or assembly 'CefSharp.Wpf, Version=110.0.300.0, Culture=neutral, PublicKeyToken=40c4b6fc221f4138' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference  
 this issue is occur me when i am version update 65 to 110 
please suggest steps

**Comment by amaitland** [comment]
There is a working example provided at https://github.com/cefsharp/CefSharp.MinimalExample I'd suggest you start there.
Pick the solution that closest matches your environment, confirm it works. You can then compare to your project.

> please suggest steps

What have you tried already?
